### DISCUSSION TITLE
Sourcing parts for the beta v 1.3.2 - solar panels issue

### DISCUSSION THREAD

**Discussion by W7DGP** [alanbjohnston/CubeSatSim#303] [General]
As I have posted before, I have finished all 3 boards and have been waiting for the solar cells in the BOM I ordered from Amazon. They were ordered Feb 8th and a ship date of Mar 7 was sent back. Today, I got quite a shocker from the solar panel vendor, uxcell. Here it is:  "Sorry to bother you. We are the customer service team from Uxcell Store.
We are sending you this message about the order you have placed on Amazon - # 113-1222527-1130624
The package of this order was shipped on Fri, Feb 9, 2024 but it has been bounced back to us.
In this case, we have issued a full refund to you for not wasting your time...... "
They took a month to tell me this? I don't know if anyone else had this problem with uxcell. 
I reordered today with a delivery date in 2 weeks. They say it shipped yesterday. We'll see. 

There is a lot to do without the solar panels like camera cable and stacking. In the middle of stacking the boards I find I ordered the wrong stacking headers from Adafruit! I have the extra tall that adds 10 mm to each stack space. It will never fit the frame so I'm ordering the correct part today. 
I can stack and test everything except solar panels. The stack is 20mm to big for the frame but that will be taken care of when the correct headers arrive next week. That will match the date for the frame pieces from the 3D printers Quickparts.com. I updated the software and all testing was OK! This was the first testing using the batteries. Its starting to get exciting.

**Comment by alanbjohnston** [comment]
Hi Larry,

That's weird.  If you don't get a ship date for that solar panel order, I'd suggest ordering with the alternate URL https://www.amazon.com/dp/B08PNNMP9Q/

Good luck!
Alan

**Reply by W7DGP** [reply]
I did get a ship date of sorts: Mar 11 to Mar 25th. 
In the mean time I have a 6 of these: AK68X37 at 5v. So I wired up a pair and tested each axis and they all work. 
Just now, (Fri PM Mar 1st) the frame pieces arrived. All I need to finish now is get the solar panels and stacking headers. The Cube Sat seems to be working reasonably well with the exception of the sim.cfg issue. I just sent the requested info.
